src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {

```

Comment:
Consider removing the empty line at the beginning of the class definition to adhere to the checklist point about not beginning class or method implementation with an empty line.